sailor/assetcentral/utils.py

```diff
@@ -429,28 +429,36 @@ def _is_non_string_iterable(obj):
     return isinstance(obj, Iterable)
 
 
+def validate_user_input(input_dict, field_map, forbidden_fields=()):
+    for field_name in forbidden_fields:
+        their_name_put = field_map.get(field_name).their_name_put
+        offender = field_name if field_name in input_dict else None
+        offender = their_name_put if their_name_put in input_dict else offender
```

Comment:
Very corner case, but if the user tries to set both our name and the AC name in the input_dict, the error message will only mention their name, right? Maybe not necessary to fix that, though

Reply:
It's fine I think.